docs/guides/02-using-style-helpers.md

```diff
@@ -0,0 +1,231 @@
+## Using style helpers
+
+In this guide, you will learn how to create insightful visualizations with few lines of code using the Web SDK style helpers. It is recommended to have read the Getting Started guide to understand the basic structure of a Web SDK application.
+
+After completing this guide, you will be able to use style helpers to create stunning visualizations in a very easy way!
+
+### Introduction
+
+The style helpers are methods inside the `carto.viz.style` namespace that allows you to visualize your layers using different properties like size or color. All the style helpers have predefined defaults for the style options that will be applied if they are not specified.
+
+This is the list of currently available style helpers:
+
+* `carto.viz.style.basic`
+* Color style helpers
+  * `carto.viz.style.colorBins`
+  * `carto.viz.style.colorCategories`
+  * `carto.viz.style.colorContinuous`  
+* Size style helpers
+  * `carto.viz.style.sizeBins` 
+  * `carto.viz.style.sizeCategories`
+  * `carto.viz.style.sizeContinuous`
+
+In the following sections we will show how to use the different style helpers.
+
+### Basic style
+
+The basic style helper allows to specify these options through the `BasicOptionsStyle` object:
+
+* color
+* size (only for points and lines)
+* opacity
+* strokeColor (only for points and polygons)
+* strokeWidth (only for points and polygons)
+
+Each of these options have different defaults depending on the type of geometry. For instance, the default color for points is #EE4D5A, #4CC8A3 for lines and #826DBA for polygons. For a complete list of all the default values for the different properties, please go to the [reference](ToDo Add link).
+
+If we use the basic style helper without specifying options, we will obtain the same result that if we were not using the helper.
```

Comment:
picky thing: if we use from the beginning "you will learn", "you xxx", I would keep the voice here using "If you use..." / "You will obtain"

Reply:
Fixed